Update hardware_acceleration.md - Intel wording changes and addition of drm hwaccel in Raspberry Pi section. (#6413)
blakeblackshear/frigate: docs/docs/configuration/hardware_acceleration.md

ffmpeg:
  hwaccel_args: preset-rpi-64-h264
```

> ##### H.264 and H.265 support
> **NOTICE**: Raspberry Pi 3 does not support hardware acceleration of H.265. H.265 can use a large amount of compute resources on the Raspberry Pi 3 and 4, in some cases it might be better to use H.264 streams.

Reviewer: ```suggestion

:::note

Raspberry Pi 3 does not support hardware acceleration of H.265. 

H.265 can use a large amount of compute resources on the Raspberry Pi 3 and 4, in some cases it might be better to use H.264 streams.

:::

```

> ```yaml
> ffmpeg:
>   hwaccel_args: -hwaccel drm
> ```

Reviewer: ```suggestion
```

Reviewer: We don't do raw args in the docs anymore, only presets.

### Intel-based CPUs

> #### Via VA-API

Reviewer: ```suggestion
#### Via VAAPI
```

> VA-API (Open Source Video Acceleration API) supports automatic profile selection so it will work automatically with both H.264 and H.265 streams. VA-API is recommended for all generations of Intel CPUs. Dedicated Quick Sync Video hardware for hardware-accelerated video encoding and decoding is supported with VA-API where available.

Reviewer: ```suggestion
VAAPI supports automatic profile selection so it will work automatically with both H.264 and H.265 streams. VAAPI is recommended for all generations of Intel CPUs. Dedicated Quick Sync Video hardware for hardware-accelerated video encoding and decoding is supported with VAAPI where available.
```

```yaml
ffmpeg:
  hwaccel_args: preset-vaapi
```

> **NOTICE**: With some of the processors, like the J4125, the default driver `iHD` doesn't seem to work correctly for hardware acceleration. You may need to change the driver to `i965` by adding the following environment variable `LIBVA_DRIVER_NAME=i965` to your docker-compose file or [in the frigate.yml for HA OS users](advanced.md#environment_vars).  You can use ```ls /dev/dri``` to check available drivers.

Reviewer: ```suggestion
**NOTICE**: With some of the processors, like the J4125, the default driver `iHD` doesn't seem to work correctly for hardware acceleration. You may need to change the driver to `i965` by adding the following environment variable `LIBVA_DRIVER_NAME=i965` to your docker-compose file or [in the frigate.yml for HA OS users](advanced.md#environment_vars).  You can use `ls /dev/dri` to check available drivers.
```
